pkg/controller/node/nodecontroller.go

```diff
@@ -115,7 +115,7 @@ type NodeController struct {
 	allocatorType     CIDRAllocatorType
 
 	cloud        cloudprovider.Interface
-	clusterCIDR  *net.IPNet
+	clusterCIDR  []*net.IPNet
```

Comment:
Just curious (without understanding the context)...should this be changed or still represent the single CIDR for the older API?